lib/kubernetes-deploy/runner.rb

```diff
@@ -9,6 +9,16 @@
 
 module KubernetesDeploy
   class Runner
+    def self.with_friendly_errors
+      yield
+    rescue FatalDeploymentError => error
+      KubernetesDeploy.logger.fatal <<-MSG
+#{error.class}: #{error.message}
+  #{error.backtrace && error.backtrace.join("\n  ")}
```

Comment:
Dunno, but if Bundler [does that](https://github.com/bundler/bundler/blob/master/lib/bundler/friendly_errors.rb#L92) I believe they have a reason.